src/Core/Geographic/Crs.ts

```diff
@@ -55,22 +60,22 @@ function _unitFromProj4Unit(projunit) {
     }
 }
 
-function toUnit(crs) {
+function toUnit(crs: ProjectionLike) {
     mustBeString(crs);
     switch (crs) {
         case 'EPSG:4326' : return UNIT.DEGREE;
         case 'EPSG:4978' : return UNIT.METER;
```

Comment:
We could delete these lines as the EPSG are defined (either in proj4 or in this file) and we could directly call the generic method as in `default` case to not create a specificity.

Reply:
Yeah agree.